mpxAPI.js

```diff
@@ -204,73 +204,78 @@ const jsonAPIResponseHandler = deserialize => response => {
 /**
  * Path constants that should be used when calling any of the `get`, `post`, etc.
  * functions on the mpxAPI client
- * 
+ *
  * @namespace
  */
 export const Path = {
   /**
    * Path to list a summary of all resources and their actions.
-   * 
+   *
    */
   Root: '/',
   /**
    * Path to contract resources.
-   * 
+   *
    */
   Contracts: '/contracts',
   /**
    * Path to fee receipeint resources.
-   * 
+   *
    */
   FeeRecipients: '/fee_recipients',
   /**
    * Path to fills resources.
-   * 
+   *
    */
   Fills: '/fills',
   /**
    * Path to perform jwt authentication.
-   * 
+   *
    */
   JWT: '/json_web_tokens',
   /**
    * Path to profile of current authenticated user.
-   * 
+   *
    */
   Me: '/me',
   /**
    * Path to orderbook resources.
-   * 
+   *
    */
   OrderBook: '/orderbooks',
   /**
    * Path to order resources on the API.
-   * 
+   *
    */
   Orders: '/orders',
   /**
    * Path to API settings
-   * 
+   *
    */
   Settings: '/settings',
   /**
    * Path to fetch token pairs on the API.
-   * 
+   *
    */
-  TokenPairs: '/token_pairs'
+  TokenPairs: '/token_pairs',
+  /**
+   * Path to fetch User Settings on the API.
+   *
+   */
+  UserSettings: '/user_settings'
 };
 
 /**
  * mpxAPI client namespace object.
- * 
+ *
  * Use the methods on this to to make requests
- * 
+ *
  * @example
  * ```
  * import { mpxAPI, Path } from '@marketprotocol/mpx-api-client'
- * 
+ *
  * mpxAPI.setHost('https://host-for-mpx-client');
```

Comment:
@travisdmathis can you update this url to our mainnet mpx-api url? Also help do the same in the Readme.md too.